framework/branch-key-store.md

```diff
@@ -109,6 +166,42 @@ If the [AWS KMS Configuration](#aws-kms-configuration) designates MRK ARN compat
 then two ARNs are compatible if they are equal in all parts other than the region.
 That is, they are compatible if [AWS KMS MRK Match for Decrypt](aws-kms/aws-kms-mrk-match-for-decrypt.md#implementation) returns true.
 
+If the [AWS KMS Configuration](#aws-kms-configuration) is Discovery or MRDiscovery,
+no comparison is ever made between ARNs.
+
+#### Discovery
+
+Discovery takes no additional information.
+
+The Keystore MAY use the KMS Key ARNs already
+persisted to the backing DynamoDB table,
+provided they are in records created
+with an identical Logical Keystore Name.
+
+The `VersionKey` and `CreateKey` Operations are NOT supported
+and will fail with a runtime exception.
+
+There is no Multi-Region logic with this configuration;
+if a Multi-Region Key is encountered,
+and the region in the ARN is not the region of the KMS Client,
+requests will fail with KMS Exceptions.
+
+#### MRDiscovery
+
+MRDiscovery takes an additional argument, which is a region.
+Any MRK ARN discovered will be changed to this region before use.
+
+The Keystore MAY use the KMS Key ARNs already
+persisted to the backing DynamoDB table,
+provided they are in records created
+with an identical Logical Keystore Name.
+
+The `VersionKey` and `CreateKey` Operations are NOT supported
+and will fail with a runtime exception.
+
+If a Multi-Region Key is encountered,
+the region in the ARN is replaced by the configured region.
```

Comment:
I think this is just repeating `Any MRK ARN discovered will be changed to this region before use.` and can be dropped.